src/interfaces/tests/MeasureValueFilter.spec.ts

```diff
@@ -0,0 +1,33 @@
+// (C) 2007-2019 GoodData Corporation
+import { isComparisonTypeOperator, isRangeTypeOperator } from "../MeasureValueFilter";
+import * as Operators from "../../constants/measureValueFilterOperators";
+
+describe("isComparisonTypeOperator", () => {
+    it("should be truthy when given string is a comparison type operator identifier", () => {
+        expect(isComparisonTypeOperator(Operators.GREATER_THAN)).toEqual(true);
```

Comment:
Please see my other comment on parametrized tests - I believe you shouldn't have unrelated `expects` in one test, because it makes telling why the test failed harder.